### ISSUE TITLE
possible addition: store enhancer

### ISSUE THREAD

**Issue by warren-bank** [markerikson/redux-ecosystem-links#70]
Hi. I submit a [pull-request](https://github.com/reactjs/redux/pull/2814) to redux yesterday that you closed with the explanation that the functionality is better suited for an (unofficial) store enhancer.

So, I went ahead and [wrote one](https://github.com/warren-bank/redux-store-enhancer).

Not sure if it's worthy of a mention, but I just wanted to put it out there.. maybe somebody will find it useful.

**Comment by markerikson** [comment]
Cool, glad to see you did that :)  One quick suggestion: rather than a generic name like `redux-store-enhancer`, maybe change the name to something like `redux-filter-subscriptions-enhancer` ?

**Comment by warren-bank** [comment]
do you feel that's still necessary.. even though I'm using an npm scope?

I feel a little silly prefixing all of my npm modules with the scope `@warren-bank/` ..but it does create a unique namespace.

**Comment by warren-bank** [comment]
..or are you thinking such a name is more descriptive
..having less to do with namespace collision

**Comment by markerikson** [comment]
Not thinking about the scope, but rather having a name that's reasonably descriptive.  There are, after all, dozens of Redux store enhancers already, which do a variety of different things.  "redux-store-enhancer" tells me it's an enhancer, but not what it actually _does_ .  "redux-filter-subscriptions-enhancer" tells me it's an enhancer that deals with filtering subscriptions :)

**Comment by warren-bank** [comment]
fair enough.. I'll rename the repo and republish it under the new name to npm

**Comment by warren-bank** [comment]
btw.. I really do appreciate the feedback.. I'm pretty much brand new to redux.. so there's a bit of learning curve :)
